src/fsharp/ast.fs

```diff
@@ -1462,19 +1462,32 @@ and
 
 and SynModuleSigDecls = SynModuleSigDecl list
 
-/// SynModuleOrNamespace(lid,isRec,isModule,decls,xmlDoc,attribs,SynAccess,m)
+and
+    [<Struct>]
+    SynModuleOrNamespaceKind =
+    | NamedModule
```

Comment:
nit: indent cases from the type name

Reply:
Thanks, I'm not used to indenting types to put an attribute after `and`, somehow I didn't noticed broken indentation. Will update when tests run is finished.